caffe2/operators/batch_gather_ops.cc

```diff
@@ -41,14 +37,14 @@ them in an output tensor of rank (q - 1) + (r - 1).
       [0, 2],
   ]
   OUTPUT = [
-      [1.0, 2.4],
-      [2.3, 3.6],
-      [4.5, 1.2],
+      [[1.0, 2.4],],
```

Comment:
I think it would make more  sense to make INDICES 1-d above (both examples are fine, the 1d one is more natural)